index.html

```diff
@@ -1,12 +1,259 @@
 <!DOCTYPE html>
-<html lang="uk">
+<html lang="en">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="author" content="Shevtsov" />
     <meta name="description" content="Домашка до п'ятого заняття (CSS)" />
     <meta name="keywords" content="HTML, CSS, проект, одна сторінка" />
     <title>CloudBudget</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
   </head>
-  <body></body>
+  <body class="body">
+    <header class="page_header">
+      <div>
+        <a href="/">
+          <img src="./images/Logo.svg" alt="CloudBudget" class="logo-img"
+        /></a>
+        <nav class="main-nav">
+          <ul class="ul-nav">
+            <li><a href="" class="li1">OVERVIEW</a></li>
```

Comment:
Maybe main-nav-link?

Reply:
Previously I wrote my answer. I named these elements "elements-for-nav"